server/command.go

```diff
@@ -17,6 +17,8 @@ const helpText = `* |/zoom start| - Start a zoom meeting`
 const oAuthHelpText = `* |/zoom connect| - Connect to zoom
 * |/zoom disconnect| - Disconnect from zoom`
```

Comment:
Can we modify these help text lines to include the `$` used below?

Reply:
not sure what you mean? the `|` and `$` are later replaced with `

Reply:
I guess what Michael means is to be coherent on what character we use for substitution across all strings. That would be a nice to have.